docs/USAGE-GUIDE.md

```diff
@@ -559,6 +559,8 @@ int s2n_config_add_cert_chain_and_key_to_store(struct s2n_config *config,
 
 **s2n_config_add_cert_chain_and_key_to_store** is the preferred method of associating a certificate chain and private key pair with an **s2n_config** object. At present, this may only be called once for each config object. It is not recommended to free or modify the **cert_key_pair** as any subsequent changes will be reflected in the config.
```

Comment:
I think we need to remove the line, "At present, this may only be called once for each config object."

Reply:
removed in 
9164919